tests/keras_tests/feature_networks_tests/feature_networks/lut_quantizer.py

```diff
@@ -155,11 +154,5 @@ def compare(self, quantized_model, float_model, input_x=None, quantization_info=
             self.unit_test.assertTrue(np.all(np.mod(ll.cluster_centers, 1) == 0))
 
 
-class RunKmeansTest(unittest.TestCase):
```

Comment:
To where did you move this test? (or did you remove it completely? if so - why?)

Reply:
The test was already in test_features_runner